content/changes.md

```diff
@@ -8,20 +8,13 @@ summary: 'Learn about new features, fixes, and other improvements'
 
 Learn about notable new features, fixes, and other improvements to the application!
 
-## January, 2024 
+## January 31, 2024 
 
 ### Features 
  
 - **Learnosity Partnership Integration** 
 
-    [Learnosity](https://learnosity.com/) has officially announced our partnership and integration into their platform to their custom base, via website, newsletter, and socials! 
-
-    This integration allows us to expand our reach into the L&D space, acting as an important add-on component within Learnosity's deep offering across many different learning domains. We've closed our first partner sale via iCEV, and already have other potential deals beginning to surface.       We will be presenting internally to their sales team later this month to further deepen our partnership and enable their sales team to bring more business to us.  
-
-    [Release Announcement](https://help.learnosity.com/hc/en-us/articles/14870489871133-2024-1-LTS-Release-Announcement)
-
-    [Partner Page](https://learnosity.com/company/partners/partners-directory/qualified/)
-
+    [Qualified and Learnosity Partnership: Bridging the Skills Gap with Developer Assessments](https://andela.com/blog-posts/qualified-and-learnosity-partnership-bridging-the-skills-gap-with-developer-assessments)
```

Comment:
I know Jake mentioned a simple note along these lines. But I still think it'd benefit from a little bit of context for what the feature is. Not a blocker or must-do, but I like my suggestion in https://github.com/andela-technology/qualified-docs/pull/199#discussion_r1575262933. Feel free to merge without it if you and Jake are confident with the current approach.